From 713f0fa0774ec236cc05911030b61e71a0103007 Mon Sep 17 00:00:00 2001
From: z00520697 <z00520697@notesmail.huawei.com/>
Date: Fri, 27 Oct 2023 19:24:08 +0800
Subject: [PATCH] TicketNo:DTS2020000000000 Description:test Team:OTHERS
 Feature or Bugfix:Bugfix Binary Source:No PrivateCode(Yes/No):No

Change-Id: I59433b6f6b3ee7b6d47681607ae06e7d8efc5f56
---
 interfaces/inner_api/huks_standard/main/include/hks_type.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/interfaces/inner_api/huks_standard/main/include/hks_type.h b/interfaces/inner_api/huks_standard/main/include/hks_type.h
index 2f69ac8c..b746d59e 100644
--- a/interfaces/inner_api/huks_standard/main/include/hks_type.h
+++ b/interfaces/inner_api/huks_standard/main/include/hks_type.h
@@ -557,6 +557,9 @@ enum HksSecureSignType {
     HKS_ASSIGN_ENUM_VALUE(HKS_TAG_CHALLENGE_POS, HKS_TAG_TYPE_UINT | 310) \
     HKS_ASSIGN_ENUM_VALUE(HKS_TAG_KEY_AUTH_PURPOSE, HKS_TAG_TYPE_UINT | 311) \
     HKS_ASSIGN_ENUM_VALUE(HKS_TAG_FRONT_USER_ID, HKS_TAG_TYPE_INT | 312) \
+    HKS_ASSIGN_ENUM_VALUE(HKS_TAG_BATCH_PURPOSE, HKS_TAG_TYPE_UINT | 313) \
+    HKS_ASSIGN_ENUM_VALUE(HKS_TAG_IS_BATCH_OPERATION, HKS_TAG_TYPE_BOOL | 314) \
+    HKS_ASSIGN_ENUM_VALUE(HKS_TAG_BATCH_OPERATION_TIMEOUT, HKS_TAG_TYPE_UINT | 315) \
     /* Attestation related TAG: 501 - 600 */\
     HKS_ASSIGN_ENUM_VALUE(HKS_TAG_ATTESTATION_CHALLENGE, HKS_TAG_TYPE_BYTES | 501) \
     HKS_ASSIGN_ENUM_VALUE(HKS_TAG_ATTESTATION_APPLICATION_ID, HKS_TAG_TYPE_BYTES | 502) \
-- 
2.34.1

